src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -737,4 +737,18 @@ function ConvertFrom-SddlString
             RawDescriptor = $rawSecurityDescriptor
         }
     }
-}
\ No newline at end of file
+}
+
+function Get-Uptime
+{
+    [CmdletBinding()]
+    [Alias("uptime")]

```

Comment:
There's an RFC for "weak aliases" that should cover this. Currently, aliases are the top priority, ahead of native commands. The weak alias spec allows for aliases that have the lowest priority, so a native uptime would always win. Another way to do this is to not use an alias at all: Just leave it as `get-uptime`. Systems that don't have a native `uptime` will yield to Powershell's last-ditch command search, which strips the get verb and tries to look for the noun. Systems with a native uptime will be found before the last-ditch noun search kicks in.